src/game/default/g_types.h

```diff
@@ -355,6 +357,42 @@ typedef enum {
 	HOOK_SWING // hookmod-style swing hook
 } g_hook_style_t;
 
+/**
+ * @brief Means of death.
+ */
+#define MOD_UNKNOWN					0
+#define MOD_BLASTER					1
+#define MOD_SHOTGUN					2
+#define MOD_SUPER_SHOTGUN			3
+#define MOD_MACHINEGUN				4
+#define MOD_GRENADE					5
+#define MOD_GRENADE_SPLASH			6
+#define MOD_ROCKET					7
+#define MOD_ROCKET_SPLASH			8
+#define MOD_HYPERBLASTER			9
+#define MOD_LIGHTNING				10
+#define MOD_LIGHTNING_DISCHARGE		11
+#define MOD_RAILGUN					12
+#define MOD_BFG_LASER				13
+#define MOD_BFG_BLAST				14
+#define MOD_WATER					15
+#define MOD_SLIME					16
+#define MOD_LAVA					17
+#define MOD_CRUSH					18
+#define MOD_TELEFRAG				19
+#define MOD_FALLING					20
+#define MOD_SUICIDE					21
+#define MOD_EXPLOSIVE				22
+#define MOD_TRIGGER_HURT			23
+#define MOD_HANDGRENADE				24
+#define MOD_HANDGRENADE_SPLASH		25
+#define MOD_HANDGRENADE_SUICIDE		26
+#define MOD_HANDGRENADE_KAMIKAZE	27
+#define MOD_FIREBALL				28
+#define MOD_HOOK					29
+#define MOD_ACT_OF_GOD				30
+#define MOD_FRIENDLY_FIRE			0x80
+
```

Comment:
Since this PR ends up modifying MODs, I'd do it here and then cherry pick it to the other branch when this one merges into master.